apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -481,6 +496,20 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             "Drag answer tiles into the answer area to the " + dropzone_layout + ". "
         )
 
+        if block_orientation is BlockOrientationType.HORIZONTAL:
+            if dropzone_layout != "bottom":
+                raise Exception(
+                    'The block-orientation attribute may only be "horizontal" when solution-placement is "bottom".'
+                )
```

Comment:
Is this restriction really necessary? I don't really care if a person wants to do it this way. If their problem only consists of just a few small blocks it would probably work just fine.

Reply:
Ah, fair. I didn't consider the case where there might be just a few small blocks. Removed.